examples/rtt.py

```diff
@@ -0,0 +1,68 @@
+import pylink
+import sys
+import time
+from builtins import input
+
+try:
+    import thread
+except ImportError:
+    import _thread as thread
+
+
+def read_rtt(jlink):
+    try:
```

Comment:
Could you add some comments and docstrings in this file to explain the behaviour?